CurrencyExchangeServiceWCF/CurrencyExchangeServiceWCF/Service.svc.cs

```diff
@@ -85,50 +104,70 @@ private string GetActualConversionRate(string currency)
 
                         while (reader.MoveToNextAttribute())
                         {// Read the attributes.
-                            if (reader.Value.Equals(currency))
+                            if (reader.Value.Equals(Currency))
```

Comment:
This changes is against the naming convention. Parameters should start with a lower case letter.

Reply:
fixed